ext/standard/basic_functions.stub.php

```diff
@@ -1075,13 +1116,13 @@ function hexdec(string $hex_string): int|float {}
 
 function octdec(string $octal_string): int|float {}
 
-function decbin($number): string {}
+function decbin(mixed $number): string {}
 
-function decoct($number): string {}
+function decoct(mixed $number): string {}
 
-function dechex($number): string {}
+function dechex(mixed $number): string {}
```

Comment:
Should these be made to accept `int` instead? It seems odd that they accept mixed but immediately cast it to int.

Reply:
I've no idea why I chose mixed, I definitely prefer `int` :)